app/src/main/java/memphis/myapplication/psync/ConsumerManager.java

```diff
@@ -24,6 +24,7 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
```

Comment:
If ArrayList is no longer needed then can be removed.

Reply:
Still using it for hello callback.